Chinese Academy of Sciences CNIC CSTR.js

```diff
@@ -0,0 +1,194 @@
+{
+	"translatorID": "c070e5a2-4bfd-44bb-9b3c-4be20c50d0d9",
+	"label": "Chinese Academy of Sciences CNIC CSTR",
+	"creator": "cheney",
+	"target": "^https?://www\\.cstr\\.cn",

```

Comment:
```suggestion
	"target": "^https?://www\\.cstr\\.cn/",

```

To match full domain names.